css/style.css

```diff
@@ -614,23 +622,30 @@ button.button-inline:hover {
 
 .colorselect {
 	overflow: hidden;
-	clear: both;
-	padding-top: 4px;
-	padding-left: 4px;
+	border-radius:3px;
+	flex-direction: row;
+	min-width: 240px;
+	height: 34px;
+	display: flex;
+	margin: 3px 3px 3px 0;
 }
 
 .colorselect .color {
 	opacity: 0.7;
-	width: 27px;
-	height: 27px;
-	float: left;
-	margin-right: 2px;
+	height: 100%;
+	flex-grow: 1;
 	border: none;
 }
 
 .colorselect .selected {
+	background-image: url(../../../core/img/actions/checkmark.svg);
```

Comment:
It owuld be better to move his to a new class `icon-selected` or `icon-checkmark`. Then it can be reused. Also, next two lines could be removed.

P.S. do we need `../../..`? on my setup(s), `url(/core/img/actions/checkmarks.svg)` works fine.